packages/consumer-sdk-js/package.json

```diff
@@ -24,7 +24,7 @@
     }
   },
   "dependencies": {
-    "@kwilteam/kwil-js": "0.7.1",
+    "@kwilteam/kwil-js": "0.9.0-beta.3",
```

Comment:
We should have a catalog for this, maybe?

Reply:
Done.